cloud/vmware/velocloud/restapi/custom/api.pm

```diff
@@ -415,9 +415,9 @@ sub get_links_qoe {
         query_form_post => {
             enterpriseId => int($self->{entreprise_id}),
             edgeId => $options{edge_id},
-            debug => 'false',
-            individualScores => 'false',
-            maxSamples => '15',
+            debug => \0,
```

Comment:
You cannot set directly 0 ?

Reply:
this format is based on this reference:
https://metacpan.org/pod/JSON::XS#other-references